marketplace/charts/marketplace-hooks/values.yaml

```diff
@@ -0,0 +1,2 @@
+imagePullPolicy: IfNotPresent
```

Comment:
missing `image` property here. A note that many helm charts follow convention of separate the image name into components: registry (gcr.io), repository (google/deployer), and tag.

Reply:
i don't think it's necessary to provide a default for `image` property.

in a follow up pull request, i'd like to set it to the base helm tiller deployer image, but that will require a little more leg work on our release process to associate chart releases with deployer image releases.